src/test/RippleKey_test.cpp

```diff
@@ -326,12 +334,20 @@ class RippleKey_test : public beast::unit_test::suite
                     "95103211B25FD07976C76D1BD0B205B37887F9F3799BA91402"
                     "1B40A6906723F47A78B66E141204E0123660F8C9D0B3F1263A"
                     "8119F4523EDB3FE6C594BFBA3603";
+                auto const expectedHash = kt == KeyType::secp256k1 ?
+                    "49D28003A776A7099EEEF64C35646AE4338E3D9065AE6A6A5DB"
+                    "FFE4BDAEB260E" :
+                    "3FE5058B1D802309DF7360A2155A97EF7A5E4213976E4E21D1F"
+                    "B154FDFC0BCCF";
+
                 BEAST_EXPECT(to_string(signer.getAccountID(sfAccount)) ==
                     expectedAccount);
                 BEAST_EXPECT(strHex(signer.getFieldVL(sfSigningPubKey)) ==
                     expectedPubKey);
                 BEAST_EXPECT(strHex(signer.getFieldVL(sfTxnSignature)) ==
                     expectedSignature);
+                BEAST_EXPECT(tx->getJson(0)["hash"].asString() ==
+                    expectedHash);
```

Comment:
Same as above. No need to build all the Json: `to_string(tx->getTransactionID())`